### PR TITLE
Fix: remove some unnecessary markup

### PR DESCRIPTION
**Pull request by bershanskiy**
This PR removes some unnecessary `<div>`s or replaces them with better alternatives. It touches files all-over `en-us` except for `web/api`.

### CODE REVIEW

File: files/en-us/web/html/element/cite/index.html [hunk 1]
```diff
@@ -139,11 +139,9 @@ <h2 id="Specifications">Specifications</h2>
 
 <h2 id="Browser_compatibility">Browser compatibility</h2>
 
-<div>
 <div class="hidden">The compatibility table in this page is generated from structured data. If you’d like to contribute to the data, please check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</div>
```
Comment (bershanskiy) [inline files/en-us/web/html/element/cite/index.html:142]: @chrisdavidmills I remember that at some point this text was necessary for every BCD table, but then it disappeared from MDN. Should I remove such notes when I come across them or leave them in?

**Review by chrisdavidmills — APPROVED** [review]
Changes all look good; thanks @bershanskiy !

File: files/en-us/web/html/element/cite/index.html (hunk 1 above)
Comment (chrisdavidmills) [reply files/en-us/web/html/element/cite/index.html:142]: I'm not 100% sure what we should be doing here. @ddbeck, what is the current practice here for these notes?

Comment (ddbeck) [reply files/en-us/web/html/element/cite/index.html:142]: @chrisdavidmills sorry for the delay on this. This text is no longer required for compat tables. I wouldn't add it to new pages and if you're making changes to existing pages' compat sections, you might remove it, but I wouldn't make any special effort to remove this text from existing pages.

Comment (chrisdavidmills) [reply files/en-us/web/html/element/cite/index.html:142]: @ddbeck ok, great, thanks Daniel!

**Pull request merged**